### PR TITLE
Run `mix format`

### PR DESCRIPTION
**Pull request by GregMefford**
We might as well get it over with before someone else submits the PR for us.

### CODE REVIEW

**Comment by GregMefford** [comment]
I think the `Nerves.Runtime.Init` code could use some more refactoring in general because it's kind of complicated and not clear what it's doing, but for this PR, I tried to hone in on just making the auto-format work in a way I like better.

File: lib/nerves_runtime/kernel/uevent.ex [hunk 1]
```diff
@@ -33,48 +36,53 @@ defmodule Nerves.Runtime.Kernel.UEvent do
...
     end
+
     {:noreply, s}
   end
 
   def registry(%{"action" => "add", "devpath" => devpath} = event, s) do
     attributes = Map.drop(event, ["action", "devpath"])
     scope = scope(devpath)
-    #Logger.debug "UEvent Add: #{inspect scope}"
+    # Logger.debug "UEvent Add: #{inspect scope}"
```
[the remark this answers is not in the document]
Comment (fhunleth) [reply lib/nerves_runtime/kernel/uevent.ex]: I think that it's fine to delete this. I'm pretty sure I know who added the comment and he's very capable of adding it back. :)

File: test/init_test.exs [hunk 2]
```diff
@@ -2,7 +2,7 @@ defmodule InitTest do
   use ExUnit.Case
   doctest Nerves.Runtime.Init
 
-  #alias Nerves.Runtime.Init
+  # alias Nerves.Runtime.Init
```
[the remark this answers is not in the document]
Comment (fhunleth) [reply test/init_test.exs]: I added an issue for us to fix this later on.

**Comment by fhunleth** [comment]
I went ahead and merged to avoid git conflicts in any other work.

**Pull request merged**